deployment/ccip/changeset/v1_5_1/cs_fast_transfer_token_pools.go

```diff
@@ -0,0 +1,355 @@
+package v1_5_1
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/latest/fast_transfer_token_pool_abstract"
+
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/deployergroup"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/stateview"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/stateview/evm"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+)
+
+var _ cldf.ChangeSet[FastTransferUpdateLaneConfigConfig] = FastTransferUpdateLaneConfigChangeset
```

Comment:
New changesets should be compatible with the newest API. See how https://github.com/smartcontractkit/chainlink/blob/2da27f832f727efc8877466a5a44e06cc37a0eab/deployment/ccip/changeset/v1_6/cs_add_new_chain_e2e.go#L43 is defined.

Also, we should be using the operations API for all new changesets. See https://github.com/smartcontractkit/chainlink/pull/18050 for an example.

Reply:
Sounds good will take a look 👍